src/smart-components/Message/utils/tokens/tokenize.ts

```diff
@@ -108,7 +108,5 @@ export function getWhiteSpacePreservedText(text: string): string {
   return text
     // convert any space or tab into the non-breaking space
     // to preserve the leading & trailing white spaces
-    .replace(/(?<!^)[ \t]+/g, '\u00A0')
-    // and keep the new line as well
-    .replace(/\n/g, '\n');
```

Comment:
Didn't need this even before since the new line is already being kept.

Reply:
Should we really need to use `U+00A0	NO-BREAK SPACE`; normal space have any issue?

Reply:
you mean, normal space ` ` <- this?

Reply:
can you comment it code, just in case ~

Reply:
Yeah left a comment about this https://github.com/sendbird/sendbird-uikit-react/pull/512/commits/f3f04349f67223284932524d9b03c9a5947d4a32